lore/estimators/sklearn.py

```diff
@@ -19,17 +19,21 @@
 
 
 class Base(lore.estimators.Base):
-    def __init__(self, estimator):
+    def __init__(self, estimator, eval_metric='sklearn_default', scoring_metric='sklearn_default'):
         super(Base, self).__init__()
         self.sklearn = estimator
+        self.eval_metric = eval_metric
```

Comment:
You'll need to provide defaults for these values that use models serialized pre 0.7. 
e.g.
https://github.com/instacart/lore/blob/master/lore/estimators/keras.py#L118-L131

Reply:
Good point. will do that.